src/board.ts

```diff
@@ -1,6 +1,9 @@
 import Pawn from './pieces/pawn';
+import Piece from './pieces/piece'
 
-const board: Array<number> = new Array(8);
+type Board = Array<(Array<(Piece | null)>)>
```

Comment:
Nie podoba mi się taki zapis :/ 
Może interface np. Field z tego a póżniej Field[][]

Reply:
No właśnie, potem to zapisałem w ten sposób (Piece | null)[][] .. co myślisz?